api/types/container/container_top.go

```diff
@@ -0,0 +1,21 @@
+package container
+
+// ----------------------------------------------------------------------------
+// DO NOT EDIT THIS FILE
+// This file was generated by `swagger generate operation`
+//
+// See hack/swagger-gen.sh
+// ----------------------------------------------------------------------------
+
+// ContainerTopOKBody container top o k body
```

Comment:
The comment gets a bit messed up because of how swagger generates it. I can look at fixing the comment generation.

Is that what you mean?